client/js/components/map/admin/AdminLayerList.vue

```diff
@@ -130,88 +130,89 @@
-            <div class="layout__item w-10/12 u-pl-0_5">
-              {{ Translator.trans('description') }}
-            </div><!--
-         --><div class="layout__item w-1/12 text-right">
-            <i
-              class="fa fa-eye u-mr-0_5"
-              v-tooltip="Translator.trans('explanation.gislayer.visibility')" />
-            </div><!--
-         --><div class="layout__item w-1/12 text-right">
-                {{ Translator.trans('edit') }}
-            </div>
+      <div v-if="hasPermission('feature_map_layer_visibility')">
+        <h3 class="u-mt">
+          {{ Translator.trans('map.bases') }}
+        </h3>
+        <!-- List-Head -->
+        <div class="color--grey u-mb-0_25 u-mt-0_5 u-mr-0_5">
+          <div class="c-at-item__row-icon layout__item u-pl-0">
+            <!-- DragHandler -->
+          </div><!--
+     -  -><div class="layout--flush layout__item c-at-item__row">
+              <div class="layout__item w-10/12 u-pl-0_5">
+                {{ Translator.trans('description') }}
+              </div><!--
+           --><div class="layout__item w-1/12 text-right">
+              <i
+                class="fa fa-eye u-mr-0_5"
+                v-tooltip="Translator.trans('explanation.gislayer.visibility')" />
+              </div><!--
+           --><div class="layout__item w-1/12 text-right">
+                  {{ Translator.trans('edit') }}
+              </div>
+          </div>
         </div>
-      </div>
-      <dp-draggable
-        v-if="false === this.isLoading"
-        :opts="draggableOptionsForBaseLayer"
-        v-model="currentBaseList"
-        :class="{'color--grey': false === isEditable}">
-        <dp-admin-layer-list-item
-          v-for="(item, idx) in currentBaseList"
-          :key="item.id"
-          :element="item"
-          :sorting-type="currentTab"
-          :is-loading="(false === isEditable)"
-          layer-type="base"
-          data-cy="baseMapLayerListItem"
-          :index="idx" />
-      </dp-draggable>
-      <div class="layout--flush u-mt u-mb">
-        <h3 class="layout__item w-1/3">
-          {{ Translator.trans('map.base.minimap') }}
-        </h3><!--
-     --><div class="layout__item w-2/3">
-          <select
-            class="o-form__control-select"
-            data-cy="adminLayerList:currentMinimapLayer"
-            v-model="currentMinimapLayer">
-            <option :value="{id: '', attributes: { name: 'default' }}">
-              {{ Translator.trans('selection.no') }}
-            </option>
-            <option
-              v-for="item in mapBaseList"
-              :key="item.id"
-              :value="item">
-              {{ item.attributes.name }}
-            </option>
-          </select>
+        <dp-draggable
+          v-if="false === this.isLoading"
+          :opts="draggableOptionsForBaseLayer"
+          v-model="currentBaseList"
+          :class="{'color--grey': false === isEditable}">
+          <dp-admin-layer-list-item
```

Comment:
the component name changed and was forgotten to be adjusted.
```suggestion
          <admin-layer-list-item
```
I think it can be done within this PR